### ISSUE TITLE
Signal strenght

### ISSUE THREAD

**Issue by realbeeit** [Xinyuan-LilyGO/LilyGO-T-SIM7000G#194]
Hello,
is it possible to check signal strenght and sent it for example via Thingsboard? I would like to make it visible in telemetry on my dashboard.

**Comment by DevinCarpenter** [comment]
Yes it is possible. You'll have to do the coding yourself and utilize a library such as TinyGSM. The command for getting signal strength is 'AT+CSQ' as per the AT Command Documentation manual.

**Comment by valdobb** [comment]
check out this repository https://github.com/valdobb/SIMTest/tree/main/src
